aws-lambda/aws-utils/tests/test.ts

```diff
@@ -1,14 +1,13 @@
 import * as assert from 'assert'
 import { decrypt } from '../index'
+const { env } = process
 
 describe('Test AWS utils', () => {
   it('can decrypt cipher text', async () => {
-    const plainText = await decrypt(
-      'AQICAHhAYaddUZOibOvj/rRTUvSdxqwlaiZHvOM4i7XV4DEr5wFz' +
-        'KI17osSCxRlGVxdSWP8/AAAAZjBkBgkqhkiG9w0BBwagVzBVAgEA' +
-        'MFAGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMt31oGEx/Hh78' +
-        'ky9YAgEQgCMbMNiEQgOFEM66S5GncBdexWG+jCVgY48oAMSZe051dfeVvg=='
-    )
-    assert.deepEqual(plainText, 'password')
+   if (!env.ENCRYPTED_TEXT) {
```

Comment:
Can we set a default for `ENCREYPTED_TEXT` somewhere?

Reply:
that's a really good idea, i should've thought of that